scripts/enderio/eio_armor_weapons.zs

```diff
@@ -335,96 +283,52 @@ assembler.recipeBuilder()
-	.inputs(<ore:plateEndSteel> * 5)
-	.inputs(<ore:foilPolyphenyleneSulfide> * 4)
-	.inputs(<ore:wireFineElectrum> * 8)
-    .inputs(<ore:screwEndSteel> * 4)
-    .fluidInputs([<fluid:soldering_alloy> * 576])
-	.outputs(<enderio:item_end_steel_shield>)
-	.EUt(480)
-	.duration(100)
-	.buildAndRegister();
+recipes.addShaped("eio_shield_end", <enderio:item_end_steel_shield>,
+	[[<ore:plateEndSteel>, <ore:ingotEndSteel>, <ore:plateEndSteel>],
+	[<ore:ingotEndSteel>, circuits[3], <ore:ingotEndSteel>],
+	[<ore:screwEndSteel>, <ore:gregScrewDrivers>, <ore:screwEndSteel>]]);
 
 // Ender Bow
-assembler.recipeBuilder()
-	.inputs(circuits[3] * 2)
-	.inputs(<ore:stickInvar> * 2)
-	.inputs(emitters[3])
-	.inputs(<ore:foilPolyphenyleneSulfide> * 4)
-	.inputs(<ore:wireFineElectrum> * 10)
-    .inputs(<ore:screwEndSteel> * 4)
-    .fluidInputs([<fluid:soldering_alloy> * 576])
-	.outputs(<enderio:item_end_steel_bow>)
-	.EUt(480)
-	.duration(100)
-	.buildAndRegister();
+recipes.addShaped("eio_bow_end", <enderio:item_end_steel_bow>,
+	[[null, <ore:string>, <ore:plateEndSteel>],
+	[<ore:string>, circuits[3], <ore:itemInfinityRod>],
+	[null, <ore:string>, <ore:plateEndSteel>]]);
 
 // Ender Pickaxe
-assembler.recipeBuilder()
-	.circuit(0)
-	.inputs(circuits[3] * 2)
-	.inputs(<ore:plateEndSteel> * 3)
-	.inputs(<ore:stickStainlessSteel> * 2)
-	.inputs(<ore:foilPolyphenyleneSulfide> * 4)
-	.inputs(<ore:wireFineElectrum> * 10)
-    .inputs(<ore:screwEndSteel> * 4)
-    .fluidInputs([<fluid:soldering_alloy> * 576])
-	.outputs(<enderio:item_end_steel_pickaxe>)
-	.EUt(480)
-	.duration(100)
-	.buildAndRegister();
+recipes.addShaped("eio_pickaxe_end", <enderio:item_end_steel_pickaxe>,
+	[[<ore:plateEndSteel>, <ore:ingotEndSteel>, <ore:ingotEndSteel>],
+	[circuits[3], <ore:itemInfinityRod>, <ore:gregScrewDrivers>],
+	[<ore:screwEndSteel>, <ore:itemInfinityRod>, <ore:screwEndSteel>]]);
 
 // Ender Axe
-assembler.recipeBuilder()
-	.circuit(1)
-	.inputs(circuits[3] * 2)
-	.inputs(<ore:plateEndSteel> * 3)
-	.inputs(<ore:foilPolyphenyleneSulfide> * 4)
-	.inputs(<ore:wireFineElectrum> * 10)
-    .inputs(<ore:screwEndSteel> * 4)
-    .fluidInputs([<fluid:soldering_alloy> * 576])
-	.outputs(<enderio:item_end_steel_axe>)
-	.EUt(480)
-	.duration(100)
-	.buildAndRegister();
+recipes.addShaped("eio_axe_end", <enderio:item_end_steel_axe>,
+	[[<ore:plateEndSteel>, <ore:ingotEndSteel>, circuits[3]],
+	[<ore:plateEndSteel>, <ore:itemInfinityRod>, <ore:gregScrewDrivers>],
+	[<ore:screwEndSteel>, <ore:itemInfinityRod>, <ore:screwEndSteel>]]);
 
 # Stellar Alloy(Steel?) Armor and Arms
 
 // Stellar Helmet
 assembler.recipeBuilder()
 	.inputs(circuits[5] * 2)
 	.inputs(<ore:plateStellarAlloy> * 5)
-	.inputs(<ore:foilPolybenzimidazole> * 5)
-	.inputs(<ore:wireFineIridium> * 10)
     .inputs(<ore:screwStellarAlloy> * 4)
-    .inputs(field_generators[6])
-    .fluidInputs([<fluid:soldering_alloy> * 576])
+    .inputs(field_generators[4])
+	.notConsumable(scripts.gregtech_globals.intCircuit(5))
```

Comment:
In config circuits for the stellar armor can be changed to 0-3 (inclusive) instead of these offset numbers now.